src/commands/basemaps-mapsheet/create-mapsheet.ts

```diff
@@ -72,30 +73,30 @@ export const basemapsCreateMapSheet = command({
   args: CommandCreateMapSheetArgs,
   async handler(args) {
     registerCli(this, args);
-    const path = args.path;
+    const url = args.path;
     const config = args.bmConfig;
-    const outputPath = args.output;
+    const outputUrl = args.output;
 
     const include = args.include ? new RegExp(args.include.toLowerCase(), 'i') : undefined;
     const exclude = args.exclude ? new RegExp(args.exclude.toLowerCase(), 'i') : undefined;
 
-    logger.info({ path }, 'MapSheet:LoadFgb');
-    const buf = await fsa.read(path);
+    logger.info({ path: url }, 'MapSheet:LoadFgb');
+    const buf = await fsa.read(url);
     logger.info({ config }, 'MapSheet:LoadConfig');
     const configJson = await readConfig(config);
     const mem = ConfigProviderMemory.fromJson(configJson);
 
     const rest = fgb.deserialize(buf) as FeatureCollection;
-    const featuresWritePromise = fsa.write('features.json', JSON.stringify(rest));
+    const featuresWritePromise = fsa.write(new URL('file://features.json'), JSON.stringify(rest));
```

Comment:
does `file://` write to the current folder?

Reply:
From the [Wikipedia article](https://en.wikipedia.org/wiki/File_URI_scheme#Number_of_slash_characters):

> -   The character sequence of two slash characters (//) after the string _file:_ denotes that either a hostname or the literal term _localhost_ follows,[\[3\]](https://en.wikipedia.org/wiki/File_URI_scheme#cite_note-3) although this part may be omitted entirely, or may contain an empty hostname.[\[4\]](https://en.wikipedia.org/wiki/File_URI_scheme#cite_note-4)
> -   The single slash between _**host**_ and _**path**_ denotes the start of the local-path part of the URI and must be present.[\[5\]](https://en.wikipedia.org/wiki/File_URI_scheme#cite_note-5)
> -   A valid file URI must therefore begin with either `file:/path` (no hostname), `file:///path` (empty hostname), or `file://hostname/path`.
> -   `file://path` (i.e. two slashes, without a hostname) is never correct, but is often used.
> -   Further slashes in _**path**_ separate directory names in a hierarchical system of directories and subdirectories. In this usage, the slash is a general, system-independent way of separating the parts, and in a particular host system it might be used as such in any pathname (as in Unix systems).

I thought `file://path` meant a relative path, while the final slash in file:///path indicated an absolute path, but it looks like that's wrong. We should *not* use `file://path` at all, and all paths should be absolute. I'll fix it.